.gitignore

```diff
@@ -1,5 +1,22 @@
-.idea/
+.DS_Store
 
-packaged.yaml
+__pycache__/
+*.py[cod]
```

Comment:
what about to switch to a WL model instead of explicit BL?

I mean it could be easier and safer to add `.*` and a few "exceptions" allowing for the hidden files or dirs you want to be committed instead that having to explicitly deny tons of hidden files ..

What do you think?